extensions/downloads-manager/src/reveal-latest-download.tsx

```diff
@@ -0,0 +1,21 @@
+import { showHUD, closeMainWindow } from "@raycast/api";
+import { getDownloads, revealFile } from "./utils";
+
+export default async function main() {
+  const downloads = getDownloads();
+
+  if (downloads.length < 1) {
+    await showHUD("No downloads found");
+    return;
+  }
+
+  const latestDownload = downloads[0];
+
+  try {
+    revealFile(latestDownload.path);
```

Comment:
You should be able to use our built-in [showInFinder](https://developers.raycast.com/api-reference/utilities#showinfinder) function.

```suggestion
    await showInFinder(latestDownload.path);
```